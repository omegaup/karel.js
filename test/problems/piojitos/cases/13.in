<ejecucion>
	<condiciones instruccionesMaximasAEjecutar="10000000" longitudStack="65000"></condiciones>
	<mundos>
		<mundo nombre="mundo_0" ancho="100" alto="100">
			<monton x="13" y="6" zumbadores="1"></monton>
			<monton x="14" y="6" zumbadores="1"></monton>
			<monton x="15" y="6" zumbadores="1"></monton>
			<monton x="16" y="6" zumbadores="1"></monton>
			<monton x="17" y="6" zumbadores="1"></monton>
			<monton x="18" y="6" zumbadores="1"></monton>
			<monton x="19" y="6" zumbadores="1"></monton>
			<monton x="20" y="6" zumbadores="1"></monton>
			<pared x1="20" y1="0" y2="1"></pared>
			<pared x1="1" y1="1" y2="2"></pared>
			<pared x1="2" y1="1" y2="2"></pared>
			<pared x1="3" y1="1" y2="2"></pared>
			<pared x1="4" y1="1" y2="2"></pared>
			<pared x1="5" y1="1" y2="2"></pared>
			<pared x1="6" y1="1" y2="2"></pared>
			<pared x1="7" y1="1" y2="2"></pared>
			<pared x1="8" y1="1" y2="2"></pared>
			<pared x1="9" y1="1" y2="2"></pared>
			<pared x1="10" y1="1" y2="2"></pared>
			<pared x1="11" y1="1" y2="2"></pared>
			<pared x1="12" y1="1" y2="2"></pared>
			<pared x1="13" y1="1" y2="2"></pared>
			<pared x1="14" y1="1" y2="2"></pared>
			<pared x1="15" y1="1" y2="2"></pared>
			<pared x1="16" y1="1" y2="2"></pared>
			<pared x1="17" y1="1" y2="2"></pared>
			<pared x1="18" y1="1" y2="2"></pared>
			<pared x1="19" y1="1" y2="2"></pared>
			<pared x1="20" y1="1" y2="2"></pared>
			<pared x1="1" y1="2" y2="3"></pared>
			<pared x1="2" y1="2" y2="3"></pared>
			<pared x1="3" y1="2" y2="3"></pared>
			<pared x1="4" y1="2" y2="3"></pared>
			<pared x1="5" y1="2" y2="3"></pared>
			<pared x1="6" y1="2" y2="3"></pared>
			<pared x1="7" y1="2" y2="3"></pared>
			<pared x1="8" y1="2" y2="3"></pared>
			<pared x1="9" y1="2" y2="3"></pared>
			<pared x1="10" y1="2" y2="3"></pared>
			<pared x1="11" y1="2" y2="3"></pared>
			<pared x1="12" y1="2" y2="3"></pared>
			<pared x1="13" y1="2" y2="3"></pared>
			<pared x1="14" y1="2" y2="3"></pared>
			<pared x1="15" y1="2" y2="3"></pared>
			<pared x1="16" y1="2" y2="3"></pared>
			<pared x1="17" y1="2" y2="3"></pared>
			<pared x1="18" y1="2" y2="3"></pared>
			<pared x1="19" y1="2" y2="3"></pared>
			<pared x1="20" y1="2" y2="3"></pared>
			<pared x1="1" y1="3" y2="4"></pared>
			<pared x1="2" y1="3" y2="4"></pared>
			<pared x1="3" y1="3" y2="4"></pared>
			<pared x1="4" y1="3" y2="4"></pared>
			<pared x1="5" y1="3" y2="4"></pared>
			<pared x1="6" y1="3" y2="4"></pared>
			<pared x1="7" y1="3" y2="4"></pared>
			<pared x1="8" y1="3" y2="4"></pared>
			<pared x1="9" y1="3" y2="4"></pared>
			<pared x1="10" y1="3" y2="4"></pared>
			<pared x1="11" y1="3" y2="4"></pared>
			<pared x1="12" y1="3" y2="4"></pared>
			<pared x1="13" y1="3" y2="4"></pared>
			<pared x1="14" y1="3" y2="4"></pared>
			<pared x1="15" y1="3" y2="4"></pared>
			<pared x1="16" y1="3" y2="4"></pared>
			<pared x1="17" y1="3" y2="4"></pared>
			<pared x1="18" y1="3" y2="4"></pared>
			<pared x1="19" y1="3" y2="4"></pared>
			<pared x1="20" y1="3" y2="4"></pared>
			<pared x1="1" y1="4" y2="5"></pared>
			<pared x1="2" y1="4" y2="5"></pared>
			<pared x1="3" y1="4" y2="5"></pared>
			<pared x1="4" y1="4" y2="5"></pared>
			<pared x1="5" y1="4" y2="5"></pared>
			<pared x1="6" y1="4" y2="5"></pared>
			<pared x1="7" y1="4" y2="5"></pared>
			<pared x1="8" y1="4" y2="5"></pared>
			<pared x1="9" y1="4" y2="5"></pared>
			<pared x1="10" y1="4" y2="5"></pared>
			<pared x1="11" y1="4" y2="5"></pared>
			<pared x1="12" y1="4" y2="5"></pared>
			<pared x1="13" y1="4" y2="5"></pared>
			<pared x1="14" y1="4" y2="5"></pared>
			<pared x1="15" y1="4" y2="5"></pared>
			<pared x1="16" y1="4" y2="5"></pared>
			<pared x1="17" y1="4" y2="5"></pared>
			<pared x1="18" y1="4" y2="5"></pared>
			<pared x1="19" y1="4" y2="5"></pared>
			<pared x1="20" y1="4" y2="5"></pared>
			<pared x1="0" y1="6" x2="1"></pared>
			<pared x1="1" y1="5" y2="6"></pared>
			<pared x1="1" y1="6" x2="2"></pared>
			<pared x1="2" y1="5" y2="6"></pared>
			<pared x1="2" y1="6" x2="3"></pared>
			<pared x1="3" y1="5" y2="6"></pared>
			<pared x1="3" y1="6" x2="4"></pared>
			<pared x1="4" y1="5" y2="6"></pared>
			<pared x1="4" y1="6" x2="5"></pared>
			<pared x1="5" y1="5" y2="6"></pared>
			<pared x1="5" y1="6" x2="6"></pared>
			<pared x1="6" y1="5" y2="6"></pared>
			<pared x1="6" y1="6" x2="7"></pared>
			<pared x1="7" y1="5" y2="6"></pared>
			<pared x1="7" y1="6" x2="8"></pared>
			<pared x1="8" y1="5" y2="6"></pared>
			<pared x1="8" y1="6" x2="9"></pared>
			<pared x1="9" y1="5" y2="6"></pared>
			<pared x1="9" y1="6" x2="10"></pared>
			<pared x1="10" y1="5" y2="6"></pared>
			<pared x1="10" y1="6" x2="11"></pared>
			<pared x1="11" y1="5" y2="6"></pared>
			<pared x1="11" y1="6" x2="12"></pared>
			<pared x1="12" y1="5" y2="6"></pared>
			<pared x1="12" y1="6" x2="13"></pared>
			<pared x1="13" y1="5" y2="6"></pared>
			<pared x1="13" y1="6" x2="14"></pared>
			<pared x1="14" y1="5" y2="6"></pared>
			<pared x1="14" y1="6" x2="15"></pared>
			<pared x1="15" y1="5" y2="6"></pared>
			<pared x1="15" y1="6" x2="16"></pared>
			<pared x1="16" y1="5" y2="6"></pared>
			<pared x1="16" y1="6" x2="17"></pared>
			<pared x1="17" y1="5" y2="6"></pared>
			<pared x1="17" y1="6" x2="18"></pared>
			<pared x1="18" y1="5" y2="6"></pared>
			<pared x1="18" y1="6" x2="19"></pared>
			<pared x1="19" y1="5" y2="6"></pared>
			<pared x1="19" y1="6" x2="20"></pared>
			<pared x1="20" y1="5" y2="6"></pared>
		</mundo>
	</mundos>
	<programas tipoEjecucion="CONTINUA" intruccionesCambioContexto="1" milisegundosParaPasoAutomatico="0">
		<programa nombre="p1" ruta="{$2$}" mundoDeEjecucion="mundo_0" xKarel="1" yKarel="1" direccionKarel="ESTE" mochilaKarel="INFINITO">
			<despliega tipo="POSICION"></despliega>
			<despliega tipo="AVANZA"></despliega>
<despliega tipo="GIRA_IZQUIERDA"></despliega>
<despliega tipo="COGE_ZUMBADOR"></despliega>
<despliega tipo="DEJA_ZUMBADOR"></despliega>
		</programa>
	</programas>
</ejecucion>
