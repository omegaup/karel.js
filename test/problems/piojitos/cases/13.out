<resultados>
	<programas>
		<programa nombre="p1" resultadoEjecucion="FIN PROGRAMA">
			<karel x="13" y="6"/>
			<instrucciones avanza="119" gira_izquierda="33" coge_zumbador="0" deja_zumbador="0"/>
		</programa>
	</programas>
</resultados>

